.github/actions/make_init/action.yml

```diff
@@ -0,0 +1,70 @@
+# https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#adding-an-action-from-the-same-repository
+runs:
+  using: composite
+  steps:
+    - name: "Install NVM, Node.js, and Yarn"
+      run: |
+        # for nvm installation
+        touch ~/.bash_profile
+        if [ ! -d ~/.nvm ] ; then
+          cd ..
+          echo $HOME
+          export NVM_DIR="$HOME/.nvm"
+          mkdir $NVM_DIR
+          # install nodejs via nvm
+          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.2/install.sh | bash
+          echo "source $NVM_DIR/nvm.sh --install" >> "$HOME/.bash_profile"
+          cd -
+          source "$HOME/.nvm/nvm.sh"
+          nvm install --lts=gallium
```

Comment:
Do you consider to create [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) file instead hardcoding value in script? This file is also supported by setup-node action.